relay-event-schema/src/protocol/ourlog.rs

```diff
@@ -46,11 +37,54 @@ pub struct OurLog {
     #[metastructure(pii = "true", trim = false)]
     pub attributes: Annotated<Object<AttributeValue>>,
 
+    /// The severity number of the log.
+    #[metastructure(required = false, trim = false)]
+    #[deprecated]
+    pub severity_number: Annotated<i64>,
+
+    /// The severity text of the log.
+    #[metastructure(required = false, trim = false)]
+    #[deprecated]
+    pub severity_text: Annotated<String>,
+
+    /// The timestamp of the log.
+    #[metastructure(required = false, trim = false)]
+    #[deprecated]
+    pub timestamp_nanos: Annotated<u64>,
+
+    /// The observed timestamp of the log.
+    #[metastructure(required = false, trim = false)]
+    #[deprecated]
+    pub observed_timestamp_nanos: Annotated<u64>,
+
+    /// The trace flags of the log.
+    #[metastructure(required = false, trim = false)]
+    #[deprecated]
+    pub trace_flags: Annotated<i64>,
```

Comment:
If they are deprecated, do we need to list them in the 'schema', they would be caught by the `other` below.

Reply:
I forgot that other moved the values into the struct, thanks